techniques/aria/ARIA11.html

```diff
@@ -112,12 +112,13 @@ <h3>Procedure</h3>
             <li>Examine each element with a <a href="https://www.w3.org/TR/wai-aria/#landmark_roles">landmark role</a>.</li>
             <li>Examine whether the correct element has been used to mark up content. For example: a <code class="language-html">navigation</code> role has been used to mark up a section with navigation links, or the <code class="language-html">main</code> role is used to contain the page's main content.</li>
             <li>If a landmark region needs to have an accessible name to be exposed as a landmark, check to see that there is an accessible name.</li>
+            <li>If the same type of landmark appears multiple times on the page, check that each one is labelled meaningfully.</li>
```

Comment:
In step 3 we talk about accessible name. In step 4 we talk about "labelled meaningfully".
Should we try to make the language more consistent?

The procedure is a little atypical in its construction. We could try to make it more usual, but it's not crucial